plone/app/standardtiles/contentlisting.py

```diff
@@ -149,6 +179,20 @@ def update(self):
             self.sort_order = 'ascending'
         self.view_template = self.data.get('view_template')
 
+    @property
+    def title(self):
+        if self.data.get('show_title'):
```

Comment:
or should it explicitly be 

`if self.data.get('show_title') and self.data.get('title'):`